arkouda/dtypes.py

```diff
@@ -177,7 +204,7 @@ def isSupportedNumber(num):
 
 def _as_dtype(dt) -> np.dtype:
     if not isinstance(dt, np.dtype):
-        return np.dtype(dt)
+        return dtype(dt)
```

Comment:
Everything looked good, this was the one detail that stood out to me, just making sure you don't need the np. here.

Reply:
we don't I'm changing it to call the `ak.dtype` now which is defined as:
```python
def dtype(x):
    # we had to create our own bigint type since numpy
    # gives them dtype=object there's no np equivalent
    if (isinstance(x, str) and x == "bigint") or isinstance(x, BigInt):
        return bigint
    else:
        return np.dtype(x)
```
this way we correctly handled `ak.bigint` but everything else calls `np.dtype` like before

Reply:
That makes sense, thank you.